go.mod

```diff
@@ -3,13 +3,18 @@ module github.com/cosmos/interchain-security
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.11
+	github.com/confio/ics23/go v0.9.0
+	github.com/cosmos/cosmos-sdk v0.45.12
```

Comment:
Note for tomorrow- does this need to be 45.13?

Reply:
We override it in the replace block with 45.13 anyway

Reply:
Case it should say 13 but there's a reason why it was bundled together with the one that upgrades to 4515